lib/launcher.js

```diff
@@ -184,7 +185,7 @@ class Launcher {
         if (!this.settings) {
             throw new Error('Failed to load settings')
         }
-        if (this.state === States.RUNNING || this.state === States.STARTING) {
+        if (this.state === States.RUNNING) {
```

Comment:
Ben, what is the thinking here? Will removing this not cause issues if the `start` function is called while a container is starting?

Reply:
This needs fixing in the forge app, the "Start" option needs to be dissabled if the project is already started (or starting)

Reply:
Forge app is already broken (showing start if project already started)

Reply:
Yeah I understand however should this not be kept as an interlock in case of UI issue or timing bug / late response disabling the start button or if is called via API twice (bypassing the UI) ?

Reply:
But we need something other than "stopped" as the state when installing the npm modules as this is just going to drive user to press start, which because it's stopped is then going to try and start it twice.

Reply:
Waiting on solution to flowforge/flowforge#1040